src/types/api.ts

```diff
@@ -158,6 +158,10 @@ export type UserInfo = {
   display_name: string | null;
 };
 
+/**
+ * Represents an annotation as returned by `h` API.
```

Comment:
```suggestion
 * Represents an annotation as returned by the h API.
```

As unfortunate a name as _h_ is (almost as bad as _X_!), we usually write it this way.

Reply:
We should start spelling it `aitch` 😂